docs/.swaggo

```diff
@@ -0,0 +1,2 @@
+// TODO: read cryptosuite.LDKeyType from sdk properly
+skip github.com/TBD54566975/ssi-sdk/did.VerificationMethod
```

Comment:
this should be solved before merge, what error did you run into?

Reply:
@decentralgabe 
`ParseComment error in file {path to did.go} :cannot find type definition: cryptosuite.LDKeyType`
the above error occurred when I ran swag init cmd.

Seems cannot load LDKeyType which define inside ssi-sdk

Reply:
probably need to fix the type definition in ssi-sdk..
What do you think?

@decentralgabe

Reply:
let me take a look. do you mind if I push to the branch if I find a solution?

Reply:
@decentralgabe 
Thanks! For sure!